code/modules/admin/view_variables/view_variables.dm

```diff
@@ -80,6 +80,8 @@
 			names += V
 	sleep(0.1 SECONDS)//For some reason, without this sleep, VVing will cause client to disconnect on certain objects.
 
+	log_admin("[key_name(usr)] viewed the variables of [D].")
```

Comment:
this will spam

Reply:
no more than other admin logs, it is important for admin investigations

Reply:
resolved in discord

Reply:
thought it would message each time, logs are good